assets/src/scripts/components/graph/index.spec.js

```diff
@@ -39,9 +39,9 @@ describe('graph component', () => {
             jasmine.Ajax.install();
             store = configureStore();
             attachToNode(store, div.node(), {
-                agencycode: 'USGS',
-                siteid: '430406089232901'
-            });
+                agencyCode: 'USGS',
+                siteId: '430406089232901'
+            }, 23);
```

Comment:
What is the 23 used for? The function only has three parameters?

Reply:
Originally I put the component ID as a separate argument, but missed this one. Fixed.